nginx/default.conf

```diff
@@ -7,6 +7,14 @@ server {
   }
 }
 
+map $http_origin $allow_origin {
+    ~^https://(.*)+\.(akvo|akvotest)\.org$ $http_origin;
+}
+
+map $http_origin $allow_methods {
+    ~^https://(.*)+\.(akvo|akvotest)\.org$ "OPTIONS, HEAD, PUT, POST, GET, DELETE";
```

Comment:
I see no harm on adding it to all domains, so that we avoid the regex duplication